public/app/features/query/state/PanelQueryRunner.ts

```diff
@@ -225,6 +227,7 @@ export class PanelQueryRunner {
       dashboardId,
       publicDashboardAccessToken,
       range: timeRange,
+      fiscalYearStartMonth,
```

Comment:
Not sure about the changes in the PanelQueryRunner. There's a lot of fiscalYearStartMonth option drilling. Maybe instead we can make sure that the time range passed to PanelQueryRunner.run method has already a time range with the fiscal year applied? This way we wouldn't really need to wory about this kind of bugs in other datasources.

Reply:
The time ranges themselves seem to be calculated before and passed correctly to graphite, but when the query is ran it uses the `raw` values instead of the precalculated dates here: https://github.com/grafana/grafana/pull/51623/files#diff-c27e82d1dbdf51beeab4bc60a1546fce8a7ffec7a7905f78f4fee06375ab05cbR182

I'm not sure why exactly this is done and why we don't just use `options.range.from`

Reply:
I guess this is some old code that we are looking at here. I would ping @grafana/observability-metrics for some help with landing this PR.